config/routes.rb

```diff
@@ -9,6 +9,7 @@
   get '/signup', to: 'users#new', as: :new_user_path
 
   get  '/create_bet', to: 'bets#new', as: :new_bet
+  post '/create_bet', to: 'bets#create', as: :create_bet
```

Comment:
use the rails `:resources` syntax. Explicitly defining routes with 'get' etc should be exceptional and used only when absolutely needed